### ISSUE TITLE
add Rate Your Music support

### ISSUE THREAD

**Issue by igorlima1701** [ganigeorgiev/hide-ratings-extension#6]
https://rateyourmusic.com/

Really liked to use this in letterboxd. Thanks!
Would be really nice if this worked in https://rateyourmusic.com/ too.

**Comment by ganigeorgiev** [comment]
Thanks for the suggestion.

The company behind RYM seems to be working on a new version of their platform (Sonemic), so I'm not sure if it is worth investing the time adding it at the moment (adding a new site to the extension requires defining an additional content_script entry and maybe an optional permission in the manifest file which will lead to very slow approval process from Chrome Web Store because we are using the v2 manifest format).

I'll keep an eye on RYM/Sonemic and may consider it in future versions of the extension.

_I'm also planning a complete rewrite using manifest v3 and adding Firefox support, but at the moment I don't have too much time working on it and it may take a while._

**Comment by igorlima1701** [comment]
Hello, ganigerogiev!

Thank you for the detailed clarification! 

I noticed that the RYM is working on this Sonemic thing for quite some time already. Hope they can release this and that this extension can be added there too in the future.

I notice that someone already made a script to hide scores on RYM too. Although when I tried using on my browser, I noticed some stuff that I think it were bugs and not all scores being hidden too. 

https://rateyourmusic.com/discussion/rate-your-music/i-created-a-script-that-hides-releases-ratings-if-you-havent-rated-them-yet/1/#post_7759436 

Thank you for the response!
